osbs/build/build_requestv2.py

```diff
@@ -177,6 +182,15 @@ def render(self, validate=True):
         self.adjust_for_isolated(self.user_params.release)
         self.render_node_selectors(self.user_params.platforms.value)
 
+        # Set our environment variables
+        custom_strategy['env'].append({
+            'name': 'USER_PARAMS',
+            'value': self.user_params.to_json(),
+        })
+        # delete the ATOMIC_REACTOR_PLUGINS placeholder
+        for env in custom_strategy['env']:
+            if env['name'] == 'ATOMIC_REACTOR_PLUGINS':
+                del custom_strategy['env'][custom_strategy['env'].index(env)]
```

Comment:
Should probably `break` here to avoid some oddities with modification while iterating.

Also, you could use `enumerate` to capture the index:

```python
for index, env in custom_strategy['env']:
  if env['name'] ==  'ATOMIC_REACTOR_PLUGINS':
    del custom_strategy['env'][index]
    break
```

Reply:
Fixed.